types/ember/index.d.ts

```diff
@@ -691,169 +593,16 @@ declare module 'ember' {
              */
             namespace: Application;
         }
+        interface Initializer<T> {
+            name: string;
+            before?: string[];
+            after?: string[];
+            initialize(application: T): void;
+        }
```

Comment:
Is this supposed to be here? I thought it was moved to: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ember__engine/-private/types/initializer.d.ts

Reply:
good catch!